docs/guide/types/line.md

```diff
@@ -51,6 +51,8 @@ The following options are available for line annotations. All of these options c
 | Name | Type | [Scriptable](../options#scriptable-options) | Default
 | ---- | ---- | ---- | :----: | ----
 | [`adjustScaleRange`](#general) | `boolean` | Yes | `true`
+| [`arrowHeads`](#arrow-heads) | `{start: object, end:object}` | Yes |
+| [`backgroundShadowColor`](#styling) | [`Color`](../options#color) | Yes | `'transparent'`
```

Comment:
If this only applies to arrowHeads, then it should be nested in there.

Reply:
done